docs/known-issues/index.md

```diff
@@ -6,6 +6,40 @@ active investigation by HPE Cray and the wider service.
 
 ## Open Issues
 
+### OOM due to memory leak in libfabric (Added: 2022-02-23)
+
+There is an underlying memory leak in the version of libfabric on ArCHER2 (that comes as part of the underlying
+SLES operating system) which can cause jobs to fail with an OOM (Out Of Memory) error. This issue will be addressed
+in a future upgrade of the ARCHER2 operating system. You can workaround this issue by setting the following 
+environment variable in your job submission scripts:
+
+```
+export FI_MR_CACHE_MAX_COUNT=0
+```
+
+This may come with a performance penalty (though in many cases, we have not seen a noticeable performance impact).
+
+If you continue to see OOM errors after setting this environment variable and do not believe that your application
+should be requesting too much memory then please
+[contact the service desk](https://www.archer2.ac.uk/support-access/servicedesk.html)
+
+### Default FFTW library points to Intel Haswell version rather than AMD Rome at runtime (Added: 2022-02-23)
+
+By default, and at runtime, the standard FFTW library version (from the module `cray-fftw`) will link to a version of the
+FFTW library optimised for the Intel Haswell architecture rather than the AMD EPYC architecture. This does not cause 
+errors as the instruction set is compatible between the two architectures but may not provide optimal performance. The
+performance differences observed have been small (&lt; 5%) but if you want to ensure that applications using the `cray-fftw`
+module use the correct version of the libraries at runtime, you should add the following lines to your job submission
+script:
+
+```
+module load cray-fftw
+export LD_LIBRARY_PATH=$CRAY_LD_LIBRARY_PATH:$LD_LIBRARY_PATH
+```
+
+This will make sure that the application uses the correct libraries rather than those that are incorrectly linked from the
+default location at `/opt/cray/pe/lib64` which is encoded into the executable's `RUNPATH`.
```

Comment:
Please could you restructure this sentence? It's not quite clear what you're trying to get across.

Reply:
Rereading, I think you can skip out the "at `/opt/cray/pe/lib64` which is encoded into the executable's `RUNPATH`" part.

Reply:
I have tried to clarify

Reply:
Thanks -- it's very clear now. The "THe" at the start of the clarified section is not capitalised correctly.